### PR TITLE
Responsive layout

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/components/Filter/filter.css [hunk 1]
```diff
@@ -41,3 +40,15 @@
...
 }
+
+@media screen and (max-width: 600px) {
+  .productlisting_sidebar {
+    width: 40%;
+  }
+}
+
+@media screen and (max-width: 450px) {
+  .price-range-inp {
```
Comment (foolhardy21) [inline src/components/Filter/filter.css:51]: For most of the selectors you have used underscore`_`, but here it's hyphen`-`, so that should be consistent.

**Review by foolhardy21 — APPROVED** [review]
css looks good. Gave one suggestion.

File: src/components/Filter/filter.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/Filter/filter.css:51]: Yaah I will change those

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! 

Just a suggestion: For mobile screens, change the width of your login and signup forms between 95-100% of `vw`. Right now they are very small

**Comment by abhisheksahani-nova** [comment]
> Looks good to me!
> 
> Just a suggestion: For mobile screens, change the width of your login and signup forms between 95-100% of `vw`. Right now they are very small

Okay sure I will do that, in next PR since I will have to change my component library, because on changing the width the alignment is getting disturbed.

File: src/components/Filter/filter.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/Filter/filter.css:51]: In next PR since I will have to change component library for that

**Pull request merged**